test/passing/tests/infix_arg_grouping.ml

```diff
@@ -57,10 +57,11 @@ hvbox 0
   $ wrap "(" ")"
       ( str txt
       $ opt mt (fun _ ->
-            fmt "@ : " $ Option.call ~f:pro_t $ psp_t $ fmt "@;<1 2>" $ bdy_t
-            $ esp_t $ Option.call ~f:epi_t ) )
-  $ fmt " ->@ " $ Option.call ~f:pro_e $ psp_e $ bdy_e $ esp_e
-  $ Option.call ~f:epi_e )
+            fmt "@ : " $ Option.call ~f:pro_t $ psp_t
+            $ fmt "@;<1 2>"
+            $ bdy_t $ esp_t $ Option.call ~f:epi_t ) )
+  $ fmt " ->@ "
+  $ Option.call ~f:pro_e $ psp_e $ bdy_e $ esp_e $ Option.call ~f:epi_e )
```

Comment:
This changed because `"@;<1 2>"` and `" ->@ "` contain spaces and are no longer "trivial".